docs/serving/services/_index.md

```diff
@@ -1,6 +1,7 @@
 ---
-title: "Knative Services"
-linkTitle: "Services"
-weight: 10
+title: "Knative services"
+weight: 02
 type: "docs"
 ---
+
+Knative services are used to deploy an application. Each Knative service is defined by a route and a configuration, which have the same name as the service, and are contained in a YAML file. Each time the configuration is updated, a new revision is created. Revisions are each backed by a deployment with two [Kubernetes Services](../knative-kubernetes-services).  
```

Comment:
I'm confused by "are contained in a YAML file" -- the configuration and route are created by the service controller, and there isn't a YAML file used to define them -- their configuration is derived from the configuration of the service.

Reply:
Yeah, reading this back it's confusing, it was supposed to mean basically I think that routes and configs are their own resources / YAML files, but I'll reword it.